zkevm-circuits/src/keccak_circuit/keccak_bit.rs

```diff
@@ -985,6 +986,12 @@ fn multi_keccak<F: Field>(bytes: &[Vec<u8>], r: F) -> Vec<KeccakRow<F>> {
     for bytes in bytes {
         keccak(&mut rows, bytes, r);
     }
+    if let Some(capacity) = capacity {
+        // Pad with no data hashes to the expected capacity
+        while rows.len() < (1 + capacity * (NUM_ROUNDS + 1)) {
```

Comment:
I think it would be worth checking for the case where the inputs `bytes` require more than `capacity`, and return an error or at least `log::err`

Reply:
Agreed can't hurt to fail as early as possible, I do expect it to fail quickly after this with a not-enough-rows-error though.